### PR TITLE
[pt] Added rule ID:QUE_SE_VERBO_PART_PASSADO

### PR DESCRIPTION
**Pull request by marcoagpinto**
Heya, @susanaboatto and @p-goulart ,

After three or four days working on this rule, it is ready.

Only yesterday I noticed that there was already a rule doing a similar thing, but this one is more powerful.

I need to search for the older rule and see if it can be deleted.

Thanks!

### CODE REVIEW

**Comment by marcoagpinto** [comment]
“Lightweight”:

```
Portuguese (Portugal): 59 total matches
Portuguese (Portugal): 811129 total sentences considered
Portuguese (Portugal): ø0.00 rule matches per sentence
```

[63stable.txt](https://github.com/languagetool-org/languagetool/files/13777780/63stable.txt)

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml [hunk 1]
```diff
@@ -12209,6 +12209,99 @@ USA
...
+                </token>
+                <token>
+                    <exception postag_regexp='yes' postag='DA.+|Z0.+'/>
+                    <exception scope='next' postag_regexp='yes' postag='DA.+|CS'/>
+                </token>
+            </pattern>
+            <filter class="org.languagetool.rules.pt.AdvancedSynthesizerFilter" args="lemmaFrom:5 lemmaSelect:V.* postagFrom:2 postagSelect:NC(.)(.).* postagReplace:VMP00[\b2][\b1].*" />
+            <message>&simplify_msg;</message>
+            <suggestion>{suggestion}</suggestion>
+            <example correction="levado">O Rui calcula o tempo <marker>que se leva</marker> a percorrer cada cidade.</example>
```
Comment (p-goulart) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml:12289]: This is ungrammatical in `pt-BR`. 'O tempo levado' sounds like 'naughty time'.

**Review by p-goulart — CHANGES_REQUESTED** [review]
If it *is* a more powerful rule, then I think we should see more *positive* examples. We see many *negative* ones (also in the APs), but I'm not convinced we don't need more *semantic* constraints, as well as syntactic.

Another nitpick I have with this is that the examples are *very long*, to the point that it takes me a while to even find the specific fragment that contains the relevant construction. I'd shorten them so that they preserve the context but don't have dozens of unrelated words around.

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml (hunk 1 above)
Comment (marcoagpinto) [reply languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml:12289]: @p-goulart 

Maybe make it pt-PT only?

Comment (p-goulart) [reply languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml:12289]: If we can't find any examples of this being grammatical in `pt-BR` but we can in `pt-PT`, then I suppose we could make it dialect-specific, yes.

**Comment by marcoagpinto** [comment]
@p-goulart 
Tonight I will close this pull request and move it to pt-PT and open another pull request.

❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️

**Comment by marcoagpinto** [comment]
Moved it to pt-PT and created a new pull request.

**Pull request closed without merging**